user-guide/install-ambassador-oss.md

```diff
@@ -120,14 +125,52 @@ kubectl port-forward ambassador-3655608000-43x86 8877
 
 will then let us view the diagnostics at `http://localhost:8877/ambassador/v0/diag/`.
 
-## 6. Enable HTTPS
+### 4. Enable HTTPS
 
 The versatile HTTPS configuration of the Ambassador API Gateway lets it support various HTTPS use cases whether simple or complex.
 
 Follow our [enabling HTTPS guide](../tls-termination) to quickly enable HTTPS support for your applications.
 
 **Note that the Ambassador Edge Stack automatically enables HTTPs.** Read more about its configuration on the [Host CRD](/reference/host-crd) page.
 
+## Helm
+
+The following will install the open-source Ambassador API Gateway with Helm.
+
+The Helm chart at https://github.com/datawire/ambassador-chart installed the Ambassador Edge Stack by default. You can still install the Ambassador API Gateway with that chart.
+
+Both Helm 2 and Helm 3 are supported. To enable CRD creation in Helm 2, the `crd-install` hook is included in the CRD manifests. When installing with Helm 3, the following message will be output to `stderr`:
+```bash
+manifest_sorter.go:175: info: skipping unknown hook: "crd-install"
+```
+Since this hook is required for Helm 2 support it **IS NOT AN ERROR AND CAN BE SAFELY IGNORED**.
+
+### 1. Add the Datawire repo to your Helm repositories
+
+```
+helm repo add datawire https://www.getambassador.io
+```
+
+### 2. Install Ambassador API Gateway
+
+The Ambassador Edge Stack is installed by default. To install the Ambassador API Gateway you need to change the `image` to point to the OSS image and set `enableAES: false` in the `values.yaml` file. See below:
+```yaml
+image:
+  repositoy: quay.io/datawire/ambassador
+  tag: $version$
+
+enableAES: false
+```
+Install the chart using the `values.yaml` file:
+```
+helm install ambassador datawire/ambassador -f values.yaml
+```
+
+This can also be done with the `--set` flag:
+```
+helm install ambassador datawire/ambassador --set image.repository=quay.io/datawire/ambassador --set image.tag=$version$ --set enableAES=false
+```
+
```

Comment:
## Helm

In the following instructions, we'll install the open-source Ambassador API
Gateway with Helm. 

Although the [Helm chart](https://github.com/datawire-ambassador-chart) installs
the Ambassador Edge Stack by default, the Ambassador API Gateway is still
available for installation for both Helm 2 and Helm 3.

With Helm 2, you must enable CRD creation with the `crd-install` hook that is
included in the CRD manifests. When installing with Helm 3, the following
message will be output to `stderr`:

```bash
manifest_sorter.go:175: info: skipping unknown hook: "crd-install"
```

Because this hook is required for Helm 2 support, it **IS NOT AN ERROR AND CAN BE SAFELY IGNORED**.

**To get started on Helm:**
1. Add the Datawire repo to your Helm repositories
2. Install the Ambassador API Gateway


### 1. Add the Datawire repo to your Helm repositories

```
helm repo add datawire https://www.getambassador.io
```

### 2. Install Ambassador API Gateway

The Ambassador Edge Stack is installed by default. To install the Ambassador API
Gateway instead,  change the `image` to point to the OSS image and set
`enableAES: false` in the `values.yaml` file.

For example:

```yaml
image:
  repositoy: quay.io/datawire/ambassador
  tag: $version$
enableAES: false
```

Then, install the chart using the `values.yaml` file:

```
helm install ambassador datawire/ambassador -f values.yaml
```

You can also install the chart with the `--set` flag:

```
helm install ambassador datawire/ambassador --set image.repository=quay.io/datawire/ambassador --set image.tag=$version$ --set enableAES=false
```